ra/deployment/src/main/java/io/quarkus/artemis/jms/ra/deployment/DevServicesArtemisProcessor.java

```diff
@@ -0,0 +1,362 @@
+                    }
+                    if (devServices.get(name) == null) {
+                        compressor.closeAndDumpCaptured();
+                    }
+                } catch (Throwable t) {
+                    compressor.closeAndDumpCaptured();
+                    throw t instanceof RuntimeException ? (RuntimeException) t : new RuntimeException(t);
+                }
+            }
+        }
+
+        if (devServices.get(name) == null) {
+            return null;
+        }
+
+        // Configure the watch dog
+        if (first) {
+            first = false;
+            Runnable closeTask = () -> {
+                for (String serviceName : devServices.keySet()) {
+                    shutdownBroker(serviceName);
+                }
+                first = true;
+                devServices.clear();
+                cfgs.clear();
+            };
+            closeBuildItem.addCloseTask(closeTask, true);
+        }
+        cfgs.put(name, Objects.requireNonNull(configuration));
+        if (devServices.get(name).isOwner()) {
+            LOGGER.infof(
+                    "Dev Services for ActiveMQ Artemis and named configuration %s started",
+                    name);
+        }
+        return devServices.get(name).toBuildItem();
+    }
+
+    private static String getArtemisUrl(String name) {
+        return devServices.get(name).getConfig().get(getUrlPropertyName(name));
+    }
+
+    private static String getUrlPropertyName(String name) {
+        if (Objects.equals(DEFAULT_CONFIG_NAME, name)) {
+            return QUARKUS_ARTEMIS_URL;
+        } else {
+            return String.format(QUARKUS_ARTEMIS_NAMED_URL_TEMPLATE, name);
+        }
+    }
+
+    private static void shutdownBroker(String name) {
+        if (devServices.get(name) != null) {
+            try {
+                devServices.get(name).close();
+            } catch (Throwable e) {
+                LOGGER.error("Failed to stop the ActiveMQ Artemis broker", e);
+            } finally {
+                devServices.remove(name);
+            }
+        }
+    }
+
+    private static RunningDevService startArtemis(
+            String name,
+            DockerStatusBuildItem dockerStatusBuildItem,
+            ArtemisDevServiceCfg config,
+            LaunchModeBuildItem launchMode,
+            Optional<Duration> timeout) {
+        if (!config.devServicesEnabled) {
+            // explicitly disabled
+            LOGGER.debugf(
+                    "Not starting dev services for ActiveMQ Artemis and configuration %s, as it has been disabled in the config.",
+                    name);
+            return null;
+        }
+
+        // Check if quarkus.artemis.url is set
+        String urlPropertyName = getUrlPropertyName(name);
+        if (ConfigUtils.isPropertyPresent(urlPropertyName)) {
+            LOGGER.debugf(
+                    "Not starting dev services for ActiveMQ Artemis and configuration %s, the quarkus.artemis.url is configured.",

```

Comment:
It looks like a copy/paste issue. So `quarkus.artemis.url` should be `quarkus.ironjacamar.ra.config.connection-parameters`?